pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/LoadSheddingStrategy.java

```diff
@@ -36,4 +37,11 @@ public interface LoadSheddingStrategy {
      * @return A map from all selected bundles to the brokers on which they reside.
      */
     Multimap<String, String> findBundlesForUnloading(LoadData loadData, ServiceConfiguration conf);
+
+    /**
+     * Triggered when active broker changes.
+     *
+     * @param activeBrokers active Brokers
+     */
+    default void onBrokerChange(Set<String> activeBrokers) {}
```

Comment:
```suggestion
    default void onActiveBrokersChange(Set<String> activeBrokers) {}
```

Reply:
Is it better to introduce an event listener interface to the LoadManager? Not all load shedding strategies are required the broker's change event. So who cares about the event, then add the listener to the load manager.